contracts/NodeDelegatorETH.sol

```diff
@@ -0,0 +1,378 @@
+
+        ISSVNetwork(SSV_NETWORK_ADDRESS).deposit(address(this), operatorIds, amount, cluster);
+    }
+
+    // @dev Withdraws SSV Tokens from a SSV Cluster
+    function withdrawSSV(uint64[] memory operatorIds, uint256 amount, Cluster memory cluster) external onlyLRTManager {
+        address SSV_NETWORK_ADDRESS = lrtConfig.getContract(LRTConstants.SSV_NETWORK);
+
+        ISSVNetwork(SSV_NETWORK_ADDRESS).withdraw(operatorIds, amount, cluster);
+    }
+
+    /// @dev Registers a new validator in the SSV Cluster
+    function registerSsvValidator(
+        bytes calldata publicKey,
+        uint64[] calldata operatorIds,
+        bytes calldata sharesData,
+        uint256 amount,
+        Cluster calldata cluster
+    )
+        external
+        onlyLRTOperator
+        whenNotPaused
+    {
+        address SSV_NETWORK_ADDRESS = lrtConfig.getContract(LRTConstants.SSV_NETWORK);
+
+        ISSVNetwork(SSV_NETWORK_ADDRESS).registerValidator(publicKey, operatorIds, sharesData, amount, cluster);
+    }
+
+    /// @dev Initiates exits from validators in a SSV Cluster
+    /// The staked ETH will eventually swept to the EigenPod.
+    /// Only the Operator can call this function.
+    /// @param publicKeys Array of validator public keys
+    /// @param operatorIds The operator IDs of the SSV Cluster
+    function exitSsvValidators(
+        bytes[] calldata publicKeys,
+        uint64[] calldata operatorIds
+    )
+        external
+        onlyLRTOperator
+        whenNotPaused
+    {
+        address SSV_NETWORK_ADDRESS = lrtConfig.getContract(LRTConstants.SSV_NETWORK);
+
+        ISSVNetwork(SSV_NETWORK_ADDRESS).bulkExitValidator(publicKeys, operatorIds);
+    }
+
+    /// @dev Remove validators from the SSV Cluster.
+    /// Make sure `exitSsvValidators` is called before and all the validators have exited the Beacon chain.
+    /// If removed before the validator has exited the beacon chain will result in the validator being penalized.
+    /// Only the Operator can call this function.
+    /// @param publicKeys Array of validator public keys
+    /// @param operatorIds The operator IDs of the SSV Cluster
+    /// @param cluster The SSV cluster details including the validator count and SSV balance
+    function removeSsvValidators(
+        bytes[] calldata publicKeys,
+        uint64[] calldata operatorIds,
+        Cluster calldata cluster
+    )
+        external
+        onlyLRTOperator
+        whenNotPaused
+    {
+        address SSV_NETWORK_ADDRESS = lrtConfig.getContract(LRTConstants.SSV_NETWORK);
+
+        ISSVNetwork(SSV_NETWORK_ADDRESS).bulkRemoveValidator(publicKeys, operatorIds, cluster);
+    }
+
+    /// @dev Withdraw all ether in the EigenPod which can be from beacon consensus rewards or validator exits.
+    /// The ether will be sent to the EigenLayer's DelayedWithdrawalRouter contract where it can be claimed after seven
+    /// days.
+    function requestEthWithdrawal() external onlyLRTOperator {
```

Comment:
🟠 We have all our `onlyLRTOperator` functions also have a `whenNotPaused` so that we can lock out a badly behaved operator. This function is missing that. This function is not as important, but let's just keep it consistent.